### PR TITLE
Add EIP-2098 support to work with compact 64-byte signatures

### PR DESCRIPTION
**Pull request by k06a**
https://eips.ethereum.org/EIPS/eip-2098

### CODE REVIEW

File: src/signature.ts [hunk 1]
```diff
@@ -96,27 +96,61 @@ export const toRpcSig = function(v: BNLike, r: Buffer, s: Buffer, chainId?: BNLi
   return bufferToHex(Buffer.concat([setLengthLeft(r, 32), setLengthLeft(s, 32), toBuffer(v)]))
 }
 
+/**
+ * Convert signature parameters into the format of Compact Signature Representation (EIP-2098).
+ * @returns Signature
+ */
+ export const toRpcSigCompact = function(v: BNLike, r: Buffer, s: Buffer, chainId?: BNLike): string {
```
Comment (axic) [inline src/signature.ts:103]: I do not think this is related to the RPC, or does any `eth_` RPC method support this now?

If not, I'd suggest something like `toEIP2098CompactSig`, `toEIP2098Sig`, or `toCompactSig` (least preferred is the last as the proposal may not be widespread enough to have "compact sig" necessarily connect people with the EIP)

**Comment by holgerd77** [comment]
@k06a hi there, cool, thanks for the PR 😄 . I could imagine that we are open for inclusion (a short pre-discussion wouldn't have been hurt either though, always some chance to get rejected which then leads to some frustration).

Short initial note: this repo is not active any more and `Util` has moved over to our monorepo https://github.com/ethereumjs/ethereumjs-monorepo. Please resubmit there for review and inclusion.

Thanks!

**Comment by axic** [comment]
@holgerd77 can this repo be marked archived or there are other reasons for it to be non-archived?

**Comment by holgerd77** [comment]
@axic hmm, there is a semiish reason since we still have got the v4, v5 and v6 branches here which theoretically might need some bugfix release in the future. This hasn't happened for several months though and in doubt we can then also un-archive for a short amount of time. So will do, likely cleaner and less prone for confusion.

@k06a I will generally close here, would be great if you can resubmit to https://github.com/ethereumjs/ethereumjs-monorepo

Thanks! 😄

**Pull request closed without merging**